### PR TITLE
docs: Adding support policy and version mapping

### PR DESCRIPTION
**Pull request by sffamily**
Add product support policy and version mapping

### CODE REVIEW

**Comment by sffamily** [comment]
@tecton @kimizhu please pay attention and help review the version mapping

**Review by tecton — APPROVED** [review]
version mapping looks good to me.

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Roadmap
+The roadmap is for information only. The listed items may be added, removed, or changed due to priority change, dependency readiness and customer demands, etc.
```
Comment (timngmsft) [inline ROADMAP.md:2]: This roadmap for Teams Toolkit is provided for informational purposes. The roadmap may change as design and development progresses. For feedback on the roadmap and to add items for consideration, log a Github issue.

Comment (timngmsft) [reply ROADMAP.md:2]: What do you think about logging a github issue? Is that our process?

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+### 2022
+
+Teams Toolkit for Visual Studio Code and Teams Toolkit CLI are GA in May 2022, Teams Toolkit for Visual Studio will be GA in August 2022.
```
Comment (timngmsft) [inline ROADMAP.md:5]: GA date for Teams Toolkit for Visual Studio Code and Teams Toolkit CLI - May 2022
GA date for Teams Toolkit for Visual Studio - August 2022

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+[placeholder]
+#### Multi-tenancy Teams app development
```
Comment (timngmsft) [inline ROADMAP.md:8]: Multi-tenant applications for Teams

**Review by timngmsft — CHANGES_REQUESTED** [review]

File: ROADMAP.md (hunk 1 above)
Comment (sffamily) [reply ROADMAP.md:2]: this is a place holder file, and please ignore the current content in this ROADMAP.md, John has been working on the final version based on a Pierce's changed structure, for Monday US time. so will replace content with the final version.

For user to open github issue. it is in the last section of SUPPORT.md

File: ROADMAP.md (hunk 2 above)
Comment (sffamily) [reply ROADMAP.md:5]: content will be replaced by John's final version tomorrow, so can ignore things in ROADMAP.md

File: ROADMAP.md (hunk 3 above)
Comment (sffamily) [reply ROADMAP.md:8]: content will be replaced by John's final version tomorrow, so can ignore things in ROADMAP.md

will check and ensure this usage in the final version.

**Review by MuyangAmigo — APPROVED** [review]

**Review by zhenjiao-ms — APPROVED** [review]

**Pull request merged**